automaxlair/MaxLairInstance.py

```diff
@@ -314,17 +315,15 @@ def identify_pokemon(
             # OCRed text.
             # Note that some OCR strings omit the ability and others omit the
             # types so don't include these identifiers in these cases.
-            string_to_match = pokemon.names[self.lang]  #.split(' (')[0]
+            string_to_match = pokemon.names[self.lang]
             if ability != '':
                 string_to_match += pokemon.abilities[self.lang]
             if types != '':
                 for type_name_dict in pokemon.types:
                     string_to_match += type_name_dict[self.lang]
             if moves != '':
                 for move_name_dict in pokemon.moves:
-                    if self.lang in move_name_dict.names:
-                        #TODO some moves are not translated yet
-                        string_to_match += move_name_dict.names[self.lang]
+                    string_to_match += move_name_dict.names[self.lang]
```

Comment:
here is a real changes. Now all languages should have all moves

Reply:
I think for names and abilities I used the get() method which supports a default return value if the key isn't included in the dict. That way, the program still works (albeit not very intelligently) in languages not supported or only partially supported by PokeAPI. Is using that approach possible for moves as well? It would be something like `string_to_match += move_name_dict.get(self.lang, move.name_id)`

Reply:
One small change that I realized looking at the whole code: I find "move_name_dict" misleading because it's a Move object, not a dict. So maybe replace it with "move" and my code above will make sense. I also just realized that would overwrite another variable; maybe we can change to use "move_text" (and "name_text", "ability_text", and "types_text").

Reply:
the existing var is moves, not move. So no need to rename those